_includes/configuration/images-upload-handler.md

```diff
@@ -2,7 +2,7 @@
 
 The **images_upload_handler** option allows you to specify a function that is used to replace {{site.productname}}'s default JavaScript upload handler function with custom logic.
 
-The upload handler function takes three arguments: `blobInfo`, a `success` callback, and a `failure` callback. When this option is not set, {{site.productname}} utilizes an `XMLHttpRequest` to upload images one at a time to the server and calls the success callback with the location of the remote image.
+The upload handler function takes four arguments: `blobInfo`, a `success` callback, a `failure` callback, and progress value (between 1 and 100). When this option is not set, {{site.productname}} utilizes an `XMLHttpRequest` to upload images one at a time to the server and calls the success callback with the location of the remote image.
```

Comment:
Should `progress` be surrounded by backticks like the others? Also, should this be included in the example below?

Reply:
_May_ have been fixed by: https://github.com/tinymce/tinymce-docs/pull/1565/commits/aeb994c8e1d04b146cff03990b8617ddc4e7e0ee